pootle/apps/accounts/models.py

```diff
@@ -241,6 +242,16 @@ def delete(self, *args, **kwargs):
         if self.is_meta:
             raise ProtectedError('Cannot remove meta user instances', None)
 
+        purge = kwargs.pop("purge", False)
+
+        if purge:
+            purge_user(self)
+        else:
+            try:
+                merge_user(self, User.objects.get_nobody_user())
+            except User.DoesNotExist:
```

Comment:
The `nobody` user is an integral part of Pootle, therefore this try/except block is redundant — if for some reason this exception happened, Pootle would terribly be broken.